app-backend/batch/src/main/scala/com/azavea/rf/batch/util/S3.scala

```diff
@@ -83,6 +83,7 @@ case class S3(
     val objectRequest = (new ListObjectsRequest)
       .withBucketName(s3bucket)
       .withPrefix(s3prefix)
+      .withMaxKeys(1000)
```

Comment:
I also couldn't find anything about the default. I _assume_ that this is the default since it seems like the default everywhere else in the universe, but even when there were only 1600 keys, it still took a surprisingly long time to get them all, so I wanted to be sure.